### DISCUSSION TITLE
Can we split into individual projects ?

### DISCUSSION THREAD

**Discussion by vivek4321** [AquilaCMS/AquilaCMS#209] [Ideas]
First thing is first, It has great potential to go long. 

Is there any reason to keep everything in one project?

Will it be good if we split into multiple modular projects, in more of head less way?
1) storefront UI 
2) back office
3) swagger specs
 
4) themes as a sub-project in the store front UI

**Comment by devaquila** [comment]
Yes for sure !
No particular reason to keep it in one project, it's historical ;)
We have started a POC to migrate to monorepo to go in this way.
As soon as we have a little more progress, we will update this feed
